jarviscli/plugins/top_music.py

```diff
@@ -0,0 +1,87 @@
+from plugin import plugin, require
+import spotipy
+from spotipy.oauth2 import SpotifyClientCredentials
+
+# Set up your credentials
+client_id = '9432469329f94c449bc36b3273f7285a'
+client_secret = '35d017ceebeb4eea83cd74ff5ee091ac'
```

Comment:
Please don't put this in code. Please generate a new secret + make this secret invalid. We don't host credentials - features that need api key are allowed, but the user must provide an own api key.